.github/workflows/hil.yml

```diff
@@ -17,6 +17,7 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
+  RUST_LOG: debug
```

Comment:
I can remove this, but I think it's quite useful when things go wrong. If a different level is better, let me know.

Reply:
No strong feelings either way, only usually check the CI logs when things fail anyway so probably makes sense to keep it in, at least for now.